tests/vector_tests.rs

```diff
@@ -0,0 +1,51 @@
+extern crate korome;
+
+use korome::Vector2;
+
+#[test]
+fn feminism(){
+    let v0 = Vector2::from((3., 6.));
+    let v1 = Vector2::from([3., 6.]);
+    let something_different = Vector2(455., 1.2);
+    let nan = Vector2(4., 0./0.);
+
+    assert_eq!(v0, v1);
+    assert_eq!(v1, v0);
+    assert_eq!(something_different, something_different);
+    assert!(nan != nan);
+    assert!(nan != v0);
+    assert!(nan != v1);
```

Comment:
This should be asserting that none of the components are NAN, instead of testing equality with NAN which will never happen.